packages/fxa-settings/src/pages/Clear/index.tsx

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import AppLayout from '../../components/AppLayout';
+import { RouteComponentProps } from '@reach/router';
+
+/* This page is only used for manual and Playwright tests and does not need l10n */
+const Clear = (_: RouteComponentProps) => {
+  try {
+    localStorage.clear();
+    sessionStorage.clear();
+    document.cookie = 'tooyoung=1; expires=Thu, 01-Jan-1970 00:00:01 GMT';
```

Comment:
FWIW, this doesn't feel very React-y, but I don't mind it vs running `useEffect` and/or using a new package like `react-cookie` here since this page doesn't have components that rerender and it's not intended to really be user-facing.